### PR TITLE
Added UI to resources

### PR DESCRIPTION
**Pull request by ElkySchles**
FinalUI drawio

### CODE REVIEW

File: src/test/resources/Touroll.drawio.png
Comment (MC-Student) [inline src/test/resources/Touroll.drawio.png:1]: Good to have picture, seems like the code didn't get pushed?

Comment (elkyhalberstam) [inline src/test/resources/Touroll.drawio.png:1]: We should have a place where you can make a timeline (drop in a few images and move them around in time).

**Comment by emmyrunge** [comment]
Is there a need for an add audio feature when the text should automatically get translated into audio?

File: src/test/resources/Touroll.drawio.png
Comment (elkyhalberstam) [inline src/test/resources/Touroll.drawio.png:1]: We could have an option to upload a text file to a slide, or type in what you want directly....

**Comment by MC-Student** [comment]
> Is there a need for an add audio feature when the text should automatically get translated into audio?

I think there could still be audio added, e.g. music

**Comment by andrewoid** [comment]
You need to fix the merge conflicts.

**Comment by ElkySchles** [comment]
> You need to fix the merge conflicts.
Screenshot 2023-11-07 at 11 42 52 AM

This is the error when I try to pull main into my branch

**Comment by andrewoid** [comment]
> 
> This is the error when I try to pull main into my branch
> 
> 

Run the line of code that says merge

**Review by andrewoid — APPROVED** [review]

**Pull request merged**